spharpy/plot/_utils.py

```diff
@@ -0,0 +1,82 @@
+"""Private utility functions for plot module."""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def _add_colorbar(colorbar, fig, ax, mappable, label):
```

Comment:
This function exactly matches pyfars pf.plot._utils._add_colorbar(). So why no directly use it or just wrap it here?